js/src/settings/edit-store-address.js

```diff
@@ -23,7 +23,7 @@ import StoreAddressCard from '.~/components/contact-information/store-address-ca
 
 const learnMoreLinkId = 'contact-information-read-more';
 const learnMoreUrl =
-	'https://woocommerce.com/document/google-listings-and-ads/#contact-information';
+	'https://woocommerce.com/document/google-for-woocommerce/get-started/requirements/#contact-information';
```

Comment:
I don't know how valid this link is for the store address. This also adds the store address to the Merchant Account, but isn't made public. This also doesn't fulfil the public contact information requirement.
I think we need to either separate the contact information section into two parts or include a mention about the contact address and phone number being added to the Merchant Center account.

The two separate parts would be:
1. Contact information synced with Merchant account (phone + store address)
2. Requirement to have at least one type of public contact information on your site (contact us page)

Should the second part be on the compliance policy page: https://woocommerce.com/document/google-for-woocommerce/compliance-policy-2

Reply:
Hi @mikkamp Not sure if I fully understand this. Can you elaborate?

Reply:
I meant because it's mixing both contact details that should be on your site and public and then also contact details which should be sent to Merchant Center it's a bit confusing what part belongs to this learnMore link. But reading it again, it does still provide the required information so maybe we can just leave it for now.